benchmarks/entropy_adiabat/plugins/entropy_model.cc

```diff
@@ -182,10 +312,93 @@ namespace aspect
+                             "The extremely large default cohesion value (1e20 Pa) prevents the viscous stress from "
+                             "exceeding the yield stress. Units: \\si{\\pascal}.");
           prm.declare_entry ("Thermal conductivity", "4.7",
                              Patterns::Double (0),
                              "The value of the thermal conductivity $k$. "
                              "Units: \\si{\\watt\\per\\meter\\per\\kelvin}.");
+          prm.declare_entry ("Thermal conductivity formulation", "constant",
+                             Patterns::Selection("constant|p-T-dependent"),
+                             "Which law should be used to compute the thermal conductivity. "
+                             "The 'constant' law uses a constant value for the thermal "
+                             "conductivity. The 'p-T-dependent' formulation uses equations "
+                             "from Stackhouse et al. (2015): First-principles calculations "
+                             "of the lattice thermal conductivity of the lower mantle "
+                             "(https://doi.org/10.1016/j.epsl.2015.06.050), and Tosi et al. "
+                             "(2013): Mantle dynamics with pressure- and temperature-dependent "
+                             "thermal expansivity and conductivity "
+                             "(https://doi.org/10.1016/j.pepi.2013.02.004) to compute the "
+                             "thermal conductivity in dependence of temperature and pressure. "
+                             "The thermal conductivity parameter sets can be chosen in such a "
+                             "way that either the Stackhouse or the Tosi relations are used. "
+                             "The conductivity description can consist of several layers with "
+                             "different sets of parameters. Note that the Stackhouse "
+                             "parametrization is only valid for the lower mantle (bridgmanite).");
+          prm.declare_entry ("Thermal conductivity transition depths", "410000, 520000, 660000",
+                             Patterns::List(Patterns::Double (0.)),
+                             "A list of depth values that indicate where the transitions between "
+                             "the different conductivity parameter sets should occur in the "
+                             "'p-T-dependent' Thermal conductivity formulation (in most cases, "
+                             "this will be the depths of major mantle phase transitions). "
+                             "Units: \\si{\\meter}.");
+          prm.declare_entry ("Reference thermal conductivities", "2.47, 3.81, 3.52, 4.9",
+                             Patterns::List(Patterns::Double (0.)),
+                             "A list of base values of the thermal conductivity for each of the "
+                             "horizontal layers in the 'p-T-dependent' Thermal conductivity "
```

Comment:
Here and all the other parameter descriptions.
```suggestion
                             "horizontal layers in the 'p-T-dependent' thermal conductivity "
```